pkg/manifests/yaml/rte/role.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: rte
   namespace: default
 rules:
-- apiGroups: ["topology.node.k8s.io"]
-  resources: ["noderesourcetopologies"]
-  verbs: ["create", "update", "get", "list"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get"]
+- apiGroups: [""]
+  resources: ["pods/status"]
+  verbs: ["update"]
```

Comment:
we need this to support RTE >= v0.3.0, but we are not there yet.
It won't hurt, and the support should be coming soon, so this time I don't mind running ahead of ourselves.